src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/kafka/KafkaOppfolgingstilfellePersonServiceSpek.kt

```diff
@@ -23,19 +27,27 @@ class KafkaOppfolgingstilfellePersonServiceSpek : Spek({
         }
 
         val kafkaOppfolgingstilfellePersonService = KafkaOppfolgingstilfellePersonService(
-            database = database
+            database = database,
+            aktivitetskravVurderingService = AktivitetskravVurderingService()
         )
 
-        val kafkaOppfolgingstilfellePerson = createKafkaOppfolgingstilfellePerson()
-        val kafkaOppfolgingstilfellePersonRecord =
-            createKafkaOppfolgingstilfellePersonConsumerRecord(kafkaOppfolgingstilfellePerson)
         val kafkaOppfolgingstilfellePersonTopicPartition = createKafkaOppfolgingstilfellePersonTopicPartition()
-
         val mockKafkaConsumerOppfolgingstilfellePerson = mockk<KafkaConsumer<String, KafkaOppfolgingstilfellePerson>>()
-        every { mockKafkaConsumerOppfolgingstilfellePerson.commitSync() } returns Unit
 
         describe("${KafkaOppfolgingstilfellePersonService::class.java.simpleName}: pollAndProcessRecords") {
-            it("commits offset") {
+
+            beforeEachTest {
+                clearMocks(mockKafkaConsumerOppfolgingstilfellePerson)
+                every { mockKafkaConsumerOppfolgingstilfellePerson.commitSync() } returns Unit
+            }
+
+            it("creates AktivitetskravVurdering(NY) for oppfolgingstilfelle lasting 9 weeks(INGEN_AKTIVITET)") {
```

Comment:
Disse er vel egentlig en ende-til-ende-tester, ikke test av `KafkaOppfolgingstilfellePersonService`? Burde i så fall hele greia hete noe annet for å reflektere det? Evt. gjøre om denne til å kun være en enhetstest? 🤔

Reply:
Tenker det er en integrasjonstest/ende-til-ende-test av KafkaOppfolgingstilfellePersonService. Har vi noen navnekonvensjon for det? Har i hvert fall lyst til å beholde testing av lagring/henting fra DB her.